docs/spec/index.md

```diff
@@ -121,6 +124,10 @@ Primary Key: (`service_id`, `run_id`, `event_sequence`)
 | `end_time` | Time | Required | Identifies the time when the employee stops working this event.<br /><br />If `trip_id` is set (and `mid_trip_end` is not `1`), this corresponds to the time of the last stop of the trip in `stop_times.txt` (after applying any trip supplement). If `end_mid_trip` is `1`, this time corresponds to a stop time in the middle of the supplemented trip, when the employee stops working on the trip. Note that this time may not exactly match `stop_times.txt` `arrival_time` or `departure_time` if the employee is considered to be working for a couple minutes after the trip finishes. This field is about when the employee is working, and consumers who care about the the trip times should check `stop_times.txt` instead. |
 | `end_mid_trip` | Enum | Optional | Indicates whether the event ends at the end of the trip or in the middle of the trip (after applying any trip supplement).<br /><br />`0` (or blank) - Run event is not associated with a trip, or no information about whether the run event ends mid-trip<br />`1` - Run event ends mid-trip<br />`2` - Run event does not end mid-trip |
 
+#### `service_id` and `trip_service_id`
```

Comment:
Something I think is missing from this section (and might still be good to keep around after the rewrite to remove `trip_service_id`): One motivation for allowing the run that does a trip to be on a different service_id than the trip is that it means that you don't have to rewrite GTFS, the trips can stay on all the same service_ids, and any complexity is only encoded in the TODS runs.

Reply:
Addressed via https://github.com/jeffkessler-keolis/operational-data-standard/pull/2